cmake/projects/Boost/schemes/url_sha1_boost_library.cmake.in

```diff
@@ -318,11 +311,22 @@ if(@HUNTER_STATUS_DEBUG@)
 endif()
 
 if(CMAKE_HOST_WIN32)
-  set(bootstrap_cmd "bootstrap.bat")
   set(b2_cmd "b2")
+  if("@MSVC@")
+    file(WRITE "@HUNTER_PACKAGE_DOWNLOAD_DIR@/patched_boostrap.bat" "\
+call \"@HUNTER_MSVC_VCVARSALL@\" @HUNTER_MSVC_ARCH@\n\
+cd @HUNTER_PACKAGE_SOURCE_DIR@\n\
+bootstrap.bat")
+    set(patch_cmd "@CMAKE_COMMAND@" -E copy "@HUNTER_PACKAGE_DOWNLOAD_DIR@/patched_boostrap.bat" "@HUNTER_PACKAGE_SOURCE_DIR@/patched_boostrap.bat")
+    set(bootstrap_cmd "patched_boostrap.bat")
+  else()
+    set(patch_cmd "")
```

Comment:
It's better to use `"@CMAKE_COMMAND@" -E echo "Dummy patch command"`. Sometimes CMake doesn't like empty stages.

Reply:
I fixed this one